conf/modules.config

```
@@ -274,6 +274,10 @@ process {
     }
 
     withName: PORECHOP_ABI {
+        ext.args = [
+            params.longread_qc_predictadapters ? "--ab_initio --discard_database" :
+            params.longread_qc_adapterlist ? "--custom_adapters --discard_database": ""
```

```suggestion
            params.longread_qc_adapterlist ? "--discard_database": ""
```

This is already in the module itself now.